index.html

```diff
@@ -2,7 +2,7 @@
 <html>
 <head>
     <meta charset="UTF-8">
-    <title>AboutCode Manager</title>
+
```

Comment:
I think you can add this back now. And remove
https://github.com/nexB/aboutcode-manager/pull/202/files#diff-7a9076d6d94e62c13d641aa71f19ae8eR17
and 
https://github.com/nexB/aboutcode-manager/pull/202/files#diff-7a9076d6d94e62c13d641aa71f19ae8eR19